CHANGES.md

```diff
@@ -11,6 +11,7 @@ Fixed Issues:
 * [#2167](https://github.com/ckeditor/ckeditor-dev/issues/2167): Fixed: Matching in [Emoji](https://ckeditor.com/cke4/addon/emoji) plugin is not case insensitive.
 * [#1084](https://github.com/ckeditor/ckeditor-dev/issues/1084) Fixed: Using the "Automatic" option with [Color Button](https://ckeditor.com/cke4/addon/colorbutton) on a text with color already defined sets an invalid color value.
 * [#966](https://github.com/ckeditor/ckeditor-dev/issues/966): Fixed: Executing [`editor.destroy()`](https://docs.ckeditor.com/ckeditor4/latest/api/CKEDITOR_editor.html#destroy) during [file upload](https://docs.ckeditor.com/ckeditor4/latest/api/CKEDITOR_fileTools_uploadWidgetDefinition.html#onUploading) throws error. Thanks to [Maksim Makarevich](https://github.com/MaksimMakarevich)!
+* [#898](https://github.com/ckeditor/ckeditor-dev/issues/898): Fixed: Long image alt text protrudes into editor when image is selected.
```

Comment:
Add [Image](https://ckeditor.com/cke4/addon/image2) reference.

Reply:
I agree about moving it to proper namespace, but also I want to clarify, that it is unrelated to fake selection. It is related to _hidden_ selection, which refers to hidden elements which will be copied to clipboard on users copy action. In our case it is an alt text.

Reply:
Please, see where `hideSelection` function is used. In a unit testing level, it's used inside fake selection and should be tested in this namespace.